### ISSUE TITLE
Repetitive markup in _pages/developers.html

### ISSUE THREAD

**Issue by anuraghazra** [glugmvit/glugmvit.github.io#129]
I guess we can loop over, instead of hard coding them? :thinking: (i'm not familiar with jekyll though)

https://github.com/glugmvit/glugmvit.github.io/blob/d6ba20644d04758f6f71c01425d881d7e215462c/_pages/developers.html#L14-L55

**Comment by cseas** [comment]
You're correct. This can be done using a [Liquid for loop](https://shopify.github.io/liquid/tags/iteration/).
